### PR TITLE
Make the "kerberos-client" module compile

### PR DESCRIPTION
**Pull request by dmajda**

### CODE REVIEW

File: data/kerberos-client.yml [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-  - testsuite/tests/ReadKrb5ConfValue.ycp
-  - testsuite/tests/Write.ycp
-  - testsuite/tests/WriteKrb5ConfValue.ycp
+moves:
+  - from: "src/Kerberos.ycp"
+    to:   src/modules
+  - from: "src/kerberos*.ycp"
+    to:   src/clients
+  - from: "src/{dialogs,wizards}.ycp"
+    to:   src/include/kterberos-client
```
Comment (lslezak) [inline data/kerberos-client.yml:12]: Typo: k_t_erberos-client

File: data/kerberos-client.yml [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+  - from: "src/kerberos.desktop"
+    to:   src/desktop
+  - from: "src/kerberos.rnc"
+    to:   src/autoyast-rnc
+
+excluded:
+  # TODO: Include files, can't be compiled for now.
+  - src/include/kterberos-client/dialogs.ycp
+  - src/include/kterberos-client/wizards.ycp
```
Comment (lslezak) [inline data/kerberos-client.yml:21]: Typo: k_t_erberos-client (2x)

File: data/kerberos-client.yml (hunk 1 above)
Comment (dmajda) [inline data/kerberos-client.yml:12]: Fixed in #62, thanks for catching this.

File: data/kerberos-client.yml (hunk 2 above)
Comment (dmajda) [inline data/kerberos-client.yml:21]: Fixed in #62, thanks for catching this.

**Comment by dmajda** [comment]
Replaced by #62. I fixed two issues pointed by @lslezak there + I also added agent moves (which I forgot). Closing.

**Pull request closed without merging**